tests/020-test-shellscripts.sh

```diff
@@ -19,7 +19,7 @@ test_shellcheck() {
 	fi
 }
 
-SHELLCHECK="$(which shellcheck 2>/dev/null)"
+SHELLCHECK="$(command -v shellcheck 2>/dev/null)"
```

Comment:
This change does not need to be part of this PR. I'm not sure if it is in a separate patch... Reviewing on itHub makes it difficult to make that distinction.